scripts/install_python_toolchain.sh

```diff
@@ -35,7 +35,11 @@ function linux_patch_sigfpe_handler {
 }
 
 $PIP install --upgrade "pip"
-$PIP install -r scripts/requirements.txt
+if [[ "$USE_MINIMAL" -eq 1  ]]; then
+  $PIP install -r scripts/requirements-minimal.txt
```

Comment:
Is `requirements-minimal.txt` a subset of `requirements.txt`? If so, why not just install everything? I'm not really sure this is worth adding additional complexity.

Reply:
for example, if we want to test this pkg with no tensorflow installed so that we can make sure this minimal pkg can work without those stripped dependencies.

Reply:
ok, that's reasonable.